src/components/CheckIn/ItemsListModal.js

```diff
@@ -67,10 +71,10 @@ const ItemsListModal = ({
       <div className={css.fieldWrapper}>
         <Select
           fullWidth
-          defaultValue={ITEM_STATUS.inProcess}
+          defaultValue={STATUS_IN_PROCESS}
```

Comment:
Actually it looks like we don't need to use `defaultValue`. Status is set on piece here https://github.com/folio-org/ui-orders/blob/master/src/components/CheckIn/CheckInDetails.js#L59

Reply:
Also it will be a great thing if you could leverage an item status select into a separate component, since we have the same field in Receiving

Reply:
but yesterday we approved change for `defaultValue` to use inProgress

Reply:
Thank you for attentively explanation. fixed.

Reply:
@NikitaSedyx, yes of course, but by logic we set default value for each items in componentDidMount other component and creating default value here does noting